src/components/forms/TaggerRemoveComponent.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { FaTimesCircle } from 'react-icons';
```

Comment:
so we found a horrible bug in webpack that would be triggered by importing this way. https://github.com/coralproject/cay/pull/48

Please change this import to be:
`import FaTimesCircle from 'react-icons/lib/fa/times-circle';`